### ISSUE TITLE
Crash when app launch with Flutter 3.x

### ISSUE THREAD

**Issue by huanghg** [flutter/flutter#109833]
A crash when app launch after upgrade to Flutter 3.x
it seems like dead lock.
and i found that there is different to create thread between flutter 3.x and flutter 2.x 

<img width="584" alt="截屏2022-08-19 11 36 28" src="https://user-images.githubusercontent.com/15646845/185555077-20882c39-a83e-4932-9bcd-2b2ba9e174cc.png">

<img width="495" alt="截屏2022-08-19 11 36 59" src="https://user-images.githubusercontent.com/15646845/185555101-4e7c1035-f4b9-4fdc-b2d9-0e3a69786c74.png">

may it possible cause dead lock? 

<details>
<summary>Crash Logs</summary>

```
Incident Identifier: ECD83AD2-53D0-4FE2-A008-CFFFB881B777
Hardware Model:      iPhone11,6
Process:             MyApp [56224]
Path:                /private/var/containers/Bundle/Application/9E46C195-CEA8-4159-8B14-14D6BDADF079/MyApp.app/MyApp
Identifier:          com.abc.xyz
Version:             2.6.0 (20220727165331)
AppStoreTools:       13F100
AppVariant:          1:iPhone11,6:15
Code Type:           ARM-64 (Native)
Role:                Foreground
Parent Process:      launchd [1]
Coalition:           com.abc.xyz [1859]

Date/Time:           2022-08-08 20:05:49.4014 +0800
Launch Time:         2022-08-08 20:05:29.2008 +0800
OS Version:          iPhone OS 15.6 (19G71)
Release Type:        User
Baseband Version:    4.04.02
Report Version:      104

Exception Type:  EXC_CRASH (SIGKILL)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Termination Reason: FRONTBOARD 2343432205 
<RBSTerminateContext| domain:10 code:0x8BADF00D explanation:scene-create watchdog transgression: application<com.abc.xyz>:56224 exhausted real (wall clock) time allowance of 18.77 seconds
ProcessVisibility: Foreground
ProcessState: Running
WatchdogEvent: scene-create
WatchdogVisibility: Foreground
WatchdogCPUStatistics: (
"Elapsed total CPU time (seconds): 78.350 (user 78.350, system 0.000), 69% CPU",
"Elapsed application CPU time (seconds): 0.184, 0% CPU"
) reportType:CrashLog maxTerminationResistance:Interactive>

Triggered by Thread:  0


Kernel Triage:
VM - pmap_enter failed with resource shortage
VM - pmap_enter failed with resource shortage
VM - pmap_enter failed with resource shortage
VM - pmap_enter failed with resource shortage
VM - pmap_enter failed with resource shortage


Thread 0 name:
Thread 0 Crashed:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   libc++.1.dylib                	0x00000001f8864e64 std::__1::__assoc_sub_state::__sub_wait(std::__1::unique_lock<std::__1::mutex>&) + 56 (future.cpp:168)
4   Flutter                       	0x0000000108f826b4 0x108b90000 + 4138676
5   Flutter                       	0x0000000108f81214 0x108b90000 + 4133396
6   Flutter                       	0x0000000108f80b60 0x108b90000 + 4131680
7   Flutter                       	0x0000000108f8037c 0x108b90000 + 4129660
8   Flutter                       	0x0000000108ba10c4 0x108b90000 + 69828
9   Flutter                       	0x0000000108ba172c 0x108b90000 + 71468
....
26  UIKitCore                     	0x00000001e219fa44 -[UIApplication _handleDelegateCallbacksWithOptions:isSuspended:restoreState:] + 340 (UIApplication.m:2334)
27  UIKitCore                     	0x00000001e2386a78 -[UIApplication _callInitializationDelegatesWithActions:forCanvas:payload:fromOriginatingProcess:] + 3572 (UIApplication.m:2754)
28  UIKitCore                     	0x00000001e236f678 -[UIApplication _runWithMainScene:transitionContext:completion:] + 1180 (UIApplication.m:4491)
29  UIKitCore                     	0x00000001e21cd1a8 -[_UISceneLifecycleMultiplexer completeApplicationLaunchWithFBSScene:transitionContext:] + 152 (_UISceneLifecycleMultiplexer.m:450)
30  UIKitCore                     	0x00000001e202757c _UIScenePerformActionsWithLifecycleActionMask + 104 (_UISceneLifecycleState.m:109)
31  UIKitCore                     	0x00000001e2152090 __101-[_UISceneLifecycleMultiplexer _evalTransitionToSettings:fromSettings:forceExit:withTransitionStore:]_block_invoke + 216 (_UISceneLifecycleMultiplexer.m:563)
32  UIKitCore                     	0x00000001e2271fe0 -[_UISceneLifecycleMultiplexer _performBlock:withApplicationOfDeactivationReasons:fromReasons:] + 248 (_UISceneLifecycleMultiplexer.m:512)
33  UIKitCore                     	0x00000001e24b0c58 -[_UISceneLifecycleMultiplexer _evalTransitionToSettings:fromSettings:forceExit:withTransitionStore:] + 772 (_UISceneLifecycleMultiplexer.m:562)
34  UIKitCore                     	0x00000001e243c1e0 -[_UISceneLifecycleMultiplexer uiScene:transitionedFromState:withTransitionContext:] + 340 (_UISceneLifecycleMultiplexer.m:468)
35  UIKitCore                     	0x00000001e2028368 __186-[_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction _performActionsForUIScene:withUpdatedFBSScene:settingsDiff:fromSettings:transitionContext:lifecycleActionType:]_block... + 196 (_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction.m:73)
36  UIKitCore                     	0x00000001e20e83c0 +[BSAnimationSettings(UIKit) tryAnimatingWithSettings:actions:completion:] + 892 (BSAnimationSettings+UIKit.m:50)
37  UIKitCore                     	0x00000001e202a26c _UISceneSettingsDiffActionPerformChangesWithTransitionContext + 276 (_UISceneSettingsDiffAction.m:23)
38  UIKitCore                     	0x00000001e2114544 -[_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction _performActionsForUIScene:withUpdatedFBSScene:settingsDiff:fromSettings:transitionContext:lifecycleActionType:] + 384 (_UIWindowSceneFBSSceneTransitionContextDrivenLifecycleSettingsDiffAction.m:58)
39  UIKitCore                     	0x00000001e2599240 __64-[UIScene scene:didUpdateWithDiff:transitionContext:completion:]_block_invoke.615 + 776 (UIScene.m:1825)
40  UIKitCore                     	0x00000001e205864c -[UIScene _emitSceneSettingsUpdateResponseForCompletion:afterSceneUpdateWork:] + 256 (UIScene.m:1526)
41  UIKitCore                     	0x00000001e209bfac -[UIScene scene:didUpdateWithDiff:transitionContext:completion:] + 288 (UIScene.m:1792)
42  UIKitCore                     	0x00000001e21d1fac -[UIApplication workspace:didCreateScene:withTransitionContext:completion:] + 584 (UIApplication.m:3955)
43  UIKitCore                     	0x00000001e2112808 -[UIApplicationSceneClientAgent scene:didInitializeWithEvent:completion:] + 388 (UIApplicationSceneClientAgent.m:45)
44  FrontBoardServices            	0x00000001f1cc4294 -[FBSScene _callOutQueue_agent_didCreateWithTransitionContext:completion:] + 440 (FBSScene.m:439)
45  FrontBoardServices            	0x00000001f1cea090 __94-[FBSWorkspaceScenesClient createWithSceneID:groupID:parameters:transitionContext:completion:]_block_invoke.215 + 128 (FBSWorkspaceScenesClient.m:384)
46  FrontBoardServices            	0x00000001f1ca5308 -[FBSWorkspace _calloutQueue_executeCalloutFromSource:withBlock:] + 240 (FBSWorkspace.m:352)
47  FrontBoardServices            	0x00000001f1ca6958 __94-[FBSWorkspaceScenesClient createWithSceneID:groupID:parameters:transitionContext:completion:]_block_invoke + 372 (FBSWorkspaceScenesClient.m:383)
48  libdispatch.dylib             	0x00000001df6fca30 _dispatch_client_callout + 20 (object.m:560)
49  libdispatch.dylib             	0x00000001df7004e0 _dispatch_block_invoke_direct + 264 (queue.c:501)
50  FrontBoardServices            	0x00000001f1ca6c70 __FBSSERIALQUEUE_IS_CALLING_OUT_TO_A_BLOCK__ + 48 (FBSSerialQueue.m:157)
51  FrontBoardServices            	0x00000001f1ca6040 -[FBSSerialQueue _targetQueue_performNextIfPossible] + 220 (FBSSerialQueue.m:181)
52  FrontBoardServices            	0x00000001f1caa700 -[FBSSerialQueue _performNextFromRunLoopSource] + 28 (FBSSerialQueue.m:194)
53  CoreFoundation                	0x00000001dfab7414 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 28 (CFRunLoop.c:1972)
54  CoreFoundation                	0x00000001dfac81a0 __CFRunLoopDoSource0 + 208 (CFRunLoop.c:2016)
55  CoreFoundation                	0x00000001dfa01694 __CFRunLoopDoSources0 + 268 (CFRunLoop.c:2053)
56  CoreFoundation                	0x00000001dfa0705c __CFRunLoopRun + 828 (CFRunLoop.c:2951)
57  CoreFoundation                	0x00000001dfa1abc8 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3268)
58  GraphicsServices              	0x00000001fbb85374 GSEventRunModal + 164 (GSEvent.c:2200)
59  UIKitCore                     	0x00000001e238eb58 -[UIApplication _run] + 1100 (UIApplication.m:3511)
60  UIKitCore                     	0x00000001e2110090 UIApplicationMain + 364 (UIApplication.m:5064)
61  MyApp                         	0x00000001048d2314 main + 384 (main.m:21)
62  dyld                          	0x0000000106ef9da4 start + 520 (dyldMain.cpp:879)

Thread 1 name:
Thread 1:
0   libsystem_kernel.dylib        	0x0000000217532138 __ulock_wait + 8
1   libdispatch.dylib             	0x00000001df6fd420 _dlock_wait + 56 (lock.c:326)
2   libdispatch.dylib             	0x00000001df6fd1d4 _dispatch_thread_event_wait_slow + 56 (lock.c:366)
3   libdispatch.dylib             	0x00000001df70bf70 __DISPATCH_WAIT_FOR_QUEUE__ + 356 (lock.h:330)
4   libdispatch.dylib             	0x00000001df70bb28 _dispatch_sync_f_slow + 144 (queue.c:1774)
5   AXCoreUtilities               	0x00000001f5c9e320 AXPerformBlockSynchronouslyOnMainThread + 108 (AXConcurrencyUtilities.m:82)
6   libdispatch.dylib             	0x00000001df6fae6c _dispatch_call_block_and_release + 32 (init.c:1517)
7   libdispatch.dylib             	0x00000001df6fca30 _dispatch_client_callout + 20 (object.m:560)
8   libdispatch.dylib             	0x00000001df6ffb44 _dispatch_queue_override_invoke + 784 (inline_internal.h:2622)
9   libdispatch.dylib             	0x00000001df70e164 _dispatch_root_queue_drain + 396 (inline_internal.h:0)
10  libdispatch.dylib             	0x00000001df70e96c _dispatch_worker_thread2 + 164 (queue.c:6935)
11  libsystem_pthread.dylib       	0x0000000250c51080 _pthread_wqthread + 228 (pthread.c:2612)
12  libsystem_pthread.dylib       	0x0000000250c50e5c start_wqthread + 8

Thread 2:
0   libsystem_pthread.dylib       	0x0000000250c50e54 start_wqthread + 0

Thread 3 name:
Thread 3:
0   libsystem_kernel.dylib        	0x00000002175314a0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	0x0000000217531ae4 mach_msg + 76 (mach_msg.c:119)
2   CoreFoundation                	0x00000001dfa02d30 __CFRunLoopServiceMachPort + 372 (CFRunLoop.c:2646)
3   CoreFoundation                	0x00000001dfa071bc __CFRunLoopRun + 1180 (CFRunLoop.c:3000)
4   CoreFoundation                	0x00000001dfa1abc8 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3268)
5   Foundation                    	0x00000001e120d444 -[NSRunLoop(NSRunLoop) runMode:beforeDate:] + 236 (NSRunLoop.m:373)
6   Foundation                    	0x00000001e124ee0c -[NSRunLoop(NSRunLoop) runUntilDate:] + 92 (NSRunLoop.m:420)
7   UIKitCore                     	0x00000001e2308cc4 -[UIEventFetcher threadMain] + 524 (UIEventFetcher.m:1167)
8   Foundation                    	0x00000001e125d41c __NSThread__start__ + 808 (NSThread.m:972)
9   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
10  libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 4 name:
Thread 4:
0   libsystem_kernel.dylib        	0x0000000217532138 __ulock_wait + 8
1   libdispatch.dylib             	0x00000001df6fd420 _dlock_wait + 56 (lock.c:326)
2   libdispatch.dylib             	0x00000001df6fd1d4 _dispatch_thread_event_wait_slow + 56 (lock.c:366)
3   libdispatch.dylib             	0x00000001df70bf70 __DISPATCH_WAIT_FOR_QUEUE__ + 356 (lock.h:330)
4   libdispatch.dylib             	0x00000001df70bb28 _dispatch_sync_f_slow + 144 (queue.c:1774)
5   UIKitCore                     	0x00000001e21e8a20 __37-[_UIRemoteKeyboards startConnection]_block_invoke_3 + 136 (_UIRemoteKeyboards.m:1094)
6   CoreFoundation                	0x00000001dfa1db24 __invoking___ + 148
7   CoreFoundation                	0x00000001dfa3b610 -[NSInvocation invoke] + 468 (NSForwarding.m:3378)
8   Foundation                    	0x00000001e1229918 __NSXPCCONNECTION_IS_CALLING_OUT_TO_REPLY_BLOCK__ + 24 (NSXPCConnection.m:170)
9   Foundation                    	0x00000001e124af54 -[NSXPCConnection _decodeAndInvokeReplyBlockWithEvent:sequence:replyInfo:] + 572 (NSXPCConnection.m:316)
10  Foundation                    	0x00000001e1225460 __88-[NSXPCConnection _sendInvocation:orArguments:count:methodSignature:selector:withProxy:]_block_invoke_3 + 212 (NSXPCConnection.m:1630)
11  libxpc.dylib                  	0x0000000250c866ac _xpc_connection_reply_callout + 124 (serializer.c:119)
12  libxpc.dylib                  	0x0000000250c79bcc _xpc_connection_call_reply_async + 88 (connection.c:833)
13  libdispatch.dylib             	0x00000001df6fcab0 _dispatch_client_callout3 + 20 (object.m:586)
14  libdispatch.dylib             	0x00000001df71a214 _dispatch_mach_msg_async_reply_invoke + 340 (mach.c:2418)
15  libdispatch.dylib             	0x00000001df704000 _dispatch_lane_serial_drain + 376 (inline_internal.h:0)
16  libdispatch.dylib             	0x00000001df704cb4 _dispatch_lane_invoke + 444 (queue.c:3944)
17  libdispatch.dylib             	0x00000001df70f500 _dispatch_workloop_worker_thread + 648 (queue.c:6732)
18  libsystem_pthread.dylib       	0x0000000250c510bc _pthread_wqthread + 288 (pthread.c:2599)
19  libsystem_pthread.dylib       	0x0000000250c50e5c start_wqthread + 8

Thread 5:
0   libsystem_pthread.dylib       	0x0000000250c50e54 start_wqthread + 0

Thread 6:
0   libsystem_pthread.dylib       	0x0000000250c50e54 start_wqthread + 0

Thread 7 name:
Thread 7:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   libc++.1.dylib                	0x00000001f8864e64 std::__1::__assoc_sub_state::__sub_wait(std::__1::unique_lock<std::__1::mutex>&) + 56 (future.cpp:168)
4   Flutter                       	0x0000000108f83d1c 0x108b90000 + 4144412
5   Flutter                       	0x0000000108e95064 0x108b90000 + 3166308
6   Flutter                       	0x0000000108e98360 0x108b90000 + 3179360
7   CoreFoundation                	0x00000001dfaaa678 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 32 (CFRunLoop.c:1812)
8   CoreFoundation                	0x00000001dfa2e654 __CFRunLoopDoTimer + 1064 (CFRunLoop.c:2420)
9   CoreFoundation                	0x00000001dfa28e00 __CFRunLoopDoTimers + 320 (CFRunLoop.c:2580)
10  CoreFoundation                	0x00000001dfa074b0 __CFRunLoopRun + 1936 (CFRunLoop.c:3116)
11  CoreFoundation                	0x00000001dfa1abc8 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3268)
12  Flutter                       	0x0000000108e98230 0x108b90000 + 3179056
13  Flutter                       	0x0000000108e97c14 0x108b90000 + 3177492
14  libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
15  libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 8 name:
Thread 8:
0   libsystem_kernel.dylib        	0x00000002175314a0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	0x0000000217531ae4 mach_msg + 76 (mach_msg.c:119)
2   CoreFoundation                	0x00000001dfa02d30 __CFRunLoopServiceMachPort + 372 (CFRunLoop.c:2646)
3   CoreFoundation                	0x00000001dfa071bc __CFRunLoopRun + 1180 (CFRunLoop.c:3000)
4   CoreFoundation                	0x00000001dfa1abc8 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3268)
5   Flutter                       	0x0000000108e98230 0x108b90000 + 3179056
6   Flutter                       	0x0000000108e97c14 0x108b90000 + 3177492
7   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
8   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 9 name:
Thread 9:
0   libsystem_kernel.dylib        	0x00000002175314a0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	0x0000000217531ae4 mach_msg + 76 (mach_msg.c:119)
2   CoreFoundation                	0x00000001dfa02d30 __CFRunLoopServiceMachPort + 372 (CFRunLoop.c:2646)
3   CoreFoundation                	0x00000001dfa071bc __CFRunLoopRun + 1180 (CFRunLoop.c:3000)
4   CoreFoundation                	0x00000001dfa1abc8 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3268)
5   Flutter                       	0x0000000108e98230 0x108b90000 + 3179056
6   Flutter                       	0x0000000108e97c14 0x108b90000 + 3177492
7   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
8   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 10 name:
Thread 10:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 11 name:
Thread 11:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 12 name:
Thread 12:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 13 name:
Thread 13:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 14 name:
Thread 14:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 15 name:
Thread 15:
0   libsystem_kernel.dylib        	0x0000000217531f24 __psynch_cvwait + 8
1   libsystem_pthread.dylib       	0x0000000250c58298 _pthread_cond_wait + 1236 (pthread_cond.c:636)
2   libc++.1.dylib                	0x00000001f886428c std::__1::condition_variable::wait(std::__1::unique_lock<std::__1::mutex>&) + 28 (__threading_support:380)
3   Flutter                       	0x0000000108e929fc 0x108b90000 + 3156476
4   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
5   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8

Thread 16 name:
Thread 16:
0   libsystem_kernel.dylib        	0x0000000217532e18 kevent + 8
1   Flutter                       	0x000000010907c010 0x108b90000 + 5160976
2   Flutter                       	0x00000001090aa93c 0x108b90000 + 5351740
3   libsystem_pthread.dylib       	0x0000000250c519ac _pthread_start + 148 (pthread.c:891)
4   libsystem_pthread.dylib       	0x0000000250c50e68 thread_start + 8


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000104   x1: 0x0000000000000000   x2: 0x0000000000000000   x3: 0x0000000000000000
    x4: 0x0000000000000000   x5: 0x00000000000000a0   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x000000016b530458   x9: 0x0000000000000001  x10: 0x00000002829e7a50  x11: 0x0000000000000002
   x12: 0x0000000000000000  x13: 0x0000000000000000  x14: 0x0000000000000000  x15: 0x0000000000000000
   x16: 0x0000000000000131  x17: 0x000000023b873ac8  x18: 0x0000000000000000  x19: 0x00000002829e7a38
   x20: 0x00000002829e7a78  x21: 0x0000000106f4c660  x22: 0x0000000000000000  x23: 0x0000000000000000
   x24: 0x0000000000000000  x25: 0x0000000000000001  x26: 0x0000000000000100  x27: 0x000000013f404370
   x28: 0x00000002829e7a28   fp: 0x000000016b5304d0   lr: 0x0000000250c58298
    sp: 0x000000016b530440   pc: 0x0000000217531f24 cpsr: 0x60000000
   esr: 0x56000080  Address size fault


Binary Images:
0x1048c8000 - 0x106227fff MyApp arm64  <7445c339bde23bfaadd1ff3516b28059> /private/var/containers/Bundle/Application/9E46C195-CEA8-4159-8B14-14D6BDADF079/MyApp.app/MyApp
0x106ee0000 - 0x106f37fff dyld arm64e  <66e1fb2668f8379ba052eb8b8291b5e1> /usr/lib/dyld
0x108b90000 - 0x109353fff Flutter arm64  <1aed3005d544309887cd044022e3ec31> /private/var/containers/Bundle/Application/9E46C195-CEA8-4159-8B14-14D6BDADF079/MyApp.app/Frameworks/Flutter.framework/Flutter
0x1df6f9000 - 0x1df73efff libdispatch.dylib arm64e  <e3ea4f635d11342aaf199f58dbc8e259> /usr/lib/system/libdispatch.dylib
0x1df9fc000 - 0x1dfe51fff CoreFoundation arm64e  <6b22dd8135853be6bc77ba19810ec0f2> /System/Library/Frameworks/CoreFoundation.framework/CoreFoundation
0x1e11f4000 - 0x1e14fefff Foundation arm64e  <ee1abaf23d7137fb906715aa79528619> /System/Library/Frameworks/Foundation.framework/Foundation
0x1e1e7a000 - 0x1e3717fff UIKitCore arm64e  <137a95aada6d332cbc01e13bb9b6e317> /System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore
0x1f1c9b000 - 0x1f1d47fff FrontBoardServices arm64e  <a314664d1369370ab222a180669b4f11> /System/Library/PrivateFrameworks/FrontBoardServices.framework/FrontBoardServices
0x1f5c9c000 - 0x1f5cb4fff AXCoreUtilities arm64e  <6e9061e5962b375db134b25126d44bb2> /System/Library/PrivateFrameworks/AXCoreUtilities.framework/AXCoreUtilities
0x1f8857000 - 0x1f88bdfff libc++.1.dylib arm64e  <17cea4f6e3933beab8971bf33851a14b> /usr/lib/libc++.1.dylib
0x1fbb84000 - 0x1fbb8cfff GraphicsServices arm64e  <a094e7b51d4037c5902756d86988ec4b> /System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices
0x217530000 - 0x217565fff libsystem_kernel.dylib arm64e  <9ab02ad0348c30cb99f40979c2d47515> /usr/lib/system/libsystem_kernel.dylib
0x250c50000 - 0x250c5bfff libsystem_pthread.dylib arm64e  <a8989ffb141439dca1a81534f21923fe> /usr/lib/system/libsystem_pthread.dylib
0x250c6b000 - 0x250ca7fff libxpc.dylib arm64e  <b183213379dc3e42b704ef71f75a577e> /usr/lib/system/libxpc.dylib

EOF
```

</details>

<details>
  <summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 3.0.5, on macOS 12.4 21F79 darwin-x64, locale zh-Hans-CN)
    • Flutter version 3.0.5 at /Users/hughes/Documents/tencent/Project/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f1875d570e (5 weeks ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/hughes/Library/Android/sdk
    • Platform android-31, build-tools 30.0.3
    • ANDROID_HOME = /Users/hughes/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/hughes/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.4.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.68.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.11.0

[✓] Connected device (3 available)
    • hugheshuang (mobile) • 00008020-000569823698002E • ios            • iOS 15.4 19E241
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 12.4 21F79 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 104.0.5112.101
    ! Error: Could not locate device support files. iPhone may be running a version of iOS that is not supported by this version of
      Xcode. You can download the latest version of Xcode from the Mac App Store or developer.apple.com. (code 601)

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

</details>

**Comment by maheshmnj** [comment]
@flutter-symbolizer-bot https://github.com/flutter/flutter/issues/109833#issue-1343993630 force flutter#v3.0.5 ios release arm64

**Comment by maheshmnj** [comment]
@flutter-symbolizer-bot https://github.com/flutter/flutter/issues/109833#issue-1343993630 flutter#v3.0.5 ios release

**Comment by maheshmnj** [comment]
Looks like the bot is not responding.
cc @mraleph

**Comment by maheshmnj** [comment]
Hi @huanghg, Thanks for filing the issue. Is the crash affecting any specific set of devices? Is the crash only in production or  can be reproduced locally?

**Comment by huanghg** [comment]
> 



> Hi @huanghg, Thanks for filing the issue. Is the crash affecting any specific set of devices? Is the crash only in production or can be reproduced locally?

Hi @maheshmnj , thanks for reply.
i found that it may crash affecting any devices and any os version.
and i can not reproduce in debug mode

**Comment by maheshmnj** [comment]
Thanks for the info @huanghg, Labeling for further insights from the team.

**Comment by huanghg** [comment]
> 

@maheshmnj  sorry about this, i found that when i archived last time, flutter version was v3.0.2. 
and it seems be fixed in 3.0.5. 
so i will close this issue. 
thanks @maheshmnj  all the same

<img width="845" alt="截屏2022-08-22 11 40 00" src="https://user-images.githubusercontent.com/15646845/185854883-83e49274-38bf-4b83-a6b7-5416201e848e.png">
,
